### PR TITLE
Snake messages

### PR DESCRIPTION
**Pull request by lidldev**
# What issue is this addressing?

## What this PR does | solves
Added comments for snake example

### CODE REVIEW

File: examples/snake/main.go [hunk 1]
```diff
@@ -102,6 +102,10 @@ func (g *Game) reset() {
 }
 
 func (g *Game) Update() error {
+	//Here, in order to prevent it from suddenly going in the opposite direction in the movement logic,
```
Comment (hajimehoshi) [inline examples/snake/main.go:105]: Add a space after `//`

File: examples/snake/main.go [hunk 2]
```diff
@@ -102,6 +102,10 @@ func (g *Game) reset() {
...
+	//we block the value used to move to the opposite side with an if statement,
```
Comment (hajimehoshi) [inline examples/snake/main.go:106]: If possible, avoid using `we` (since it is not clear who 'we' are)

Comment (lidldev) [reply examples/snake/main.go:106]: Im sorry I directly copy and pasted from translate I hate translate. wait I will fix it

**Comment by hajimehoshi** [comment]
Why does PR include changes for `examples/platformer/main.go`??

**Comment by lidldev** [comment]
> Why does PR include changes for `examples/platformer/main.go`??

 I don't know It looked like this on the pull request screen, I even changed the branch.🤔

**Comment by hajimehoshi** [comment]
Let me close this PR to save GitHub workflows resources. Create a new PR when you are really ready. Thanks,

**Pull request closed without merging**